sigle/src/modules/analytics/containers/StoryItemAnalytics.tsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from 'react';
+import { toast } from 'react-toastify';
+import { SubsetStory } from '../../../types';
+import { getStoriesFile } from '../../../utils';
+import { StoryItemAnalytics as Component } from '../components/StoryItemAnalytics';
+
+interface StorItemAnalyticsProps {
+  storyId: string;
+}
+export const StoryItemAnalytics = ({ storyId }: StorItemAnalyticsProps) => {
+  const [stories, setStories] = useState<SubsetStory[] | null>(null);
+
+  const loadStoryFile = async () => {
+    try {
+      const file = await getStoriesFile();
```

Comment:
should we rather load the individual story file? Also let's please use useQuery there :)